src/types/num.rs

```diff
@@ -242,10 +232,175 @@ int_convert_u64_or_i64!(
     ffi::PyLong_AsUnsignedLongLong
 );
 
+#[cfg(all(not(Py_LIMITED_API), target_endian = "little"))]
+int_convert_128!(i128, 16, 1);
 #[cfg(not(Py_LIMITED_API))]
-int_convert_bignum!(i128, 16, IS_LITTLE_ENDIAN, 1);
-#[cfg(not(Py_LIMITED_API))]
-int_convert_bignum!(u128, 16, IS_LITTLE_ENDIAN, 0);
+int_convert_128!(u128, 16, 0);
```

Comment:
`i128` conversions on big-endian appears to have been inadvertently deleted

Reply:
No, it's at line 236

Reply:
line 236 is only enabled on little-endian systems, so it can't provide the `i128` conversions on big-endian systems since it is disabled.

Reply:
Ah, I understand.
Thank you for pointing it out!